src/airflow/airqo_etl_utils/schema/calibrated_measurements.json

```diff
@@ -0,0 +1,70 @@
+[
```

Comment:
schema for tables that will temporarily stop calibrated data during analysis and validation